packages/viewer/src/commands/streamCommands.ts

```diff
@@ -36,6 +36,7 @@ export function connect({
   };
 }
 
+// TODO(dan): remove, we can call the api directly.
```

Comment:
You planning to do anything with this in this pr? Also why would we remove this? I thought going through FSS was to reduce latency on requests.

Reply:
That comment is perhaps misleading. I meant that you can use `StreamApi` directly. I'm considering removing the concept of the `CommandRegistry` altogether. Turns out it's need has diminished over time.

Reply:
Ah, I understand. Perhaps removing the TODO, or updating the text if you choose to leave it in upon merging.

Reply:
Annotated this with `@deprecated`.